### ISSUE TITLE
dist folder not being removed

### ISSUE THREAD

**Issue by Jojoshua** [johnagan/clean-webpack-plugin#93]
Windows 10

....
 "webpack": "4.19.0",
 "clean-webpack-plugin": "0.1.19",

....
let pathsToClean = [
    'dist'
]
........
    output: {
        path: path.resolve(__dirname, "dist"),
        filename: '[name].js',
        chunkFilename: '[name].bundle.js',
    },
.......

  new CleanWebpackPlugin(pathsToClean),

The output shows that is worked logging this =>clean-webpack-plugin: {path}\dist has been removed. However all the files and folders remained. The plugin had no effect.

When i changed the following, everything was wiped out as I intended.
let pathsToClean = [
    'dist/ * . *', 
    'dist/fonts', 
    'dist/img', 
]

Is this expected behavior? I would have though just setting the configuration to clean 'dist' would be enough.

**Comment by robertmirro** [comment]
I just stepped through this in vscode. `rimraf` eventually calls `fs.rmdirSync()` which was silently failing with "_Error: EBUSY: resource busy or locked..._" because [`EBUSY` is not trapped](https://github.com/isaacs/rimraf/blob/master/rimraf.js#L327-L334).  

In my case, I had a file inside .dist open in an editor.  Closing the file resolved this issue.

**Comment by vilien** [comment]
I got same problem, and i just modify the configuration to 'dist/*'. It works.

**Comment by wanweiwang2012** [comment]
# reference project structure
\+ webpack-template
&#8194;&#8194;\+ dist
&#8194;&#8194;\+ config
&#8194;&#8194;&#8194;&#8194;\- webpack.config.js
&#8194;&#8194;+ src
&#8194;&#8194;&#8194;&#8194;- index.html
&#8194;&#8194;&#8194;&#8194;- index.js

```
new CleanWebpackPlugin(['dist'], {
  // the default is the webpack configuration file directory, please specify the project root directory
  root: path.resolve(__dirname, "/"),
  verbose: true,
  dry: false,
}),
```

**Comment by chrisblossom** [comment]
Closed in `v2.0.0` via #99. See [johnagan/clean-webpack-plugin#usage](https://github.com/johnagan/clean-webpack-plugin#usage). Please open a new issue if you continue to have problems.
